src/classes.scss

```diff
@@ -43,3 +43,4 @@
 @import './wizard/classes';
 @import './skip-to-content/classes';
 @import './surface/classes';
+@import './anchor/classes';
```

Comment:
nit: I think this file needs to be sorted alphabetically

Reply:
This has been fixed.